core/backend/src/test/integration/BriefcaseManager.test.ts

```diff
@@ -143,6 +148,14 @@ describe("BriefcaseManager (#integration)", () => {
     assert.isFalse(IModelJsFs.existsSync(pathname3));
   });
 
+  it("should log usage when reusing checkpoints", async () => {

```

Comment:
this test should somehow ensure that the checkpoint is a V1 checkpoint, that the file doesn't already exist the first time, and that it only counts once when there's a retry. Then it should test against V2 checkpoints.

BTW, this is a terrible way to log usage. Why don't we do that from DR when we start?

Reply:
> BTW, this is a terrible way to log usage. Why don't we do that from DR when we start?

@kabentley I'm working on that with the Design Review team. Hoping in 2.17.x we can roll out the removal of default usage logging in openForRead and each application will be in charge of it.

Reply:
that's good news. We should (ultimately) be able to redirect a given user to multiple backends during the course of a single session, so only the frontend knows what "usage" means.

Reply:
but, if that's the plan i wouldn't delay this PR improving the tests.

Reply:
just reverted those test changes since this is bound to be moot when Caleb removes `BriefcaseManager.logUsage`

Reply:
@kabentley the plan is to remove this auto usage logging from the IModelDb.open in 2.17, at which time in DR we'll start handling usage ourselves. We also will want a way to toggle this auto usage logging to verify our own logging leading up to the 2.17 release so it's a smooth transition.

Reply:
@jason-crow I'll add something to the 2.16 release to toggle it. That way we can test prior to the 2.17 removal.